frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -616,6 +622,63 @@ export const experimentLogic = kea<experimentLogicType>([
                 },
             },
         ],
+        requiredEventsBreakdown: [
```

Comment:
I'd much prefer rolling this into the backend API errors: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/queries/experiments/funnel_experiment_result.py#L129

for a few reasons:

1. This makes an extra unnecessary query and can be slow
2. Doesn't handle all edge cases, like when actions are present
3. We technically don't need flag information on all events: On atleast one step is sufficient.

All these are already handled by the backend, so we can adjust shape of data to give us what we want here.

Something like:

(a) Detect if the events are coming in
(b) And if no flag value on above^, change validation error to implementation_error, and pass on appropriate message in the frontend

Reply:
You have the funnel results here: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/queries/experiments/funnel_experiment_result.py#L87

which give us all the information we need about whether events are coming in and whether they have flag info associated with them. 

As long as step 1 has flag information, we're good, so we don't need to check every step, and can always link to step 1 issues.

Reply:
💯 @neilkakkar 

I'm gonna split this into two separate PRs - funnels and trends. Closing this one.